src/controllers/hid/hidiooutputreport.cpp

```diff
@@ -31,6 +31,8 @@ void HidIoOutputReport::updateCachedData(const QByteArray& data,
         const RuntimeLoggingCategory& logOutput,
         HidIoGlobalOutputReportFifo* pGlobalOutputReportFifo,
         bool useNonSkippingFIFO) {
+    Q_UNUSED(deviceInfo);
+    Q_UNUSED(pGlobalOutputReportFifo);
```

Comment:
Please remove the unused parameters from this function, instead of silencing the warnings.

Reply:
Tbh I didn't take a closer look if this are just base implementations that may be overriden.
If not, sure I can remove the arguments. (why are they there in the first place? no one cared about the warnings?)

Reply:
They are a leftover of a PR review. This information was used to print information in console messages. These message printing was removed during the review.
I didn't saw ans warnings. Shouldn't warnings be handled as errors?

Reply:
Alright, I removed the arguments.

> Shouldn't warnings be handled as errors?

Not that I'm aware of, I just assumed we try to get rid of warnings so they can easily be spotted in the build log when adding new code.